Documentation/Helm-Charts/operator-chart.gotmpl.md

```diff
@@ -36,6 +36,9 @@ helm repo add rook-release https://charts.rook.io/release
 helm install --create-namespace --namespace rook-ceph rook-ceph rook-release/rook-ceph -f values.yaml
 ```
 
+!!! Note
+    --namespace specify the operator namespace.
```

Comment:
Do you think this note is necessary? I don't see what other namespace it would be confused with, or that needs clarifying. The `--namespace` flag is a common helm flag, so it seems ok not to explain more.

Reply:
Yes right the `--namespace` is provided by helm,
But first time I really get confused as we generally use `namespace` as cephcluster namespace, 
I am fine keeping it or not keeping it, as per your suggestion.

Reply:
I vote to remove this note in the operator helm chart doc. We can keep the clarification in the other doc since the cluster helm chart has both the cluster namespace and operator namespace.